unit_tests/sources/declarative/auth/test_oauth.py

```diff
@@ -384,7 +382,7 @@ def test_refresh_access_token_expire_format(
     @freezegun.freeze_time("2020-01-01")
     def test_set_token_expiry_date_no_format(self, mocker, expires_in_response, next_day, raises):
         config.update(
-            {"token_expiry_date": pendulum.parse(next_day).subtract(days=2).to_rfc3339_string()}
+            {"token_expiry_date": (ab_datetime_parse(next_day) - timedelta(days=2)).isoformat()}
```

Comment:
Ditto. `str()` should be fine here.

```suggestion
            {"token_expiry_date": str(ab_datetime_parse(next_day) - timedelta(days=2))}
```